src/rpc/api/wire/messages.ts

```diff
@@ -0,0 +1,89 @@
+import {Slot} from "../../types/primitive";
+import {BlockRootSlot, HashTreeRoot} from "./types";
+
+type RPCMethod = Hello | Goodbye | GetStatus;
+
+export interface Request {
```

Comment:
you have these interfaces defined on two places, probably should be moved to types directory

Reply:
I have omitted this completely.